src/TBuffer.cpp

```diff
@@ -748,6 +748,7 @@ TBuffer::TBuffer(Host* pH)
 , lastloggedToLine(0)
 , mEncoding()
 , mMainIncomingCodec(nullptr)
+, mEchoingText(false)
```

Comment:
This is out of position I think - shouldn't it be right after `, mMXP_SEND_NO_REF_MODE(false)` ...!

Reply:
Adjusted.

Reply:
It's there now!